jsonschema/dataschema

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
i like this. it would be cool to write a test that validates your example datasets against this.